modules/trusted-firmware-m/Kconfig

```
@@ -206,4 +206,13 @@ config ROM_START_OFFSET
 
 endif # !TFM_BL2
 
+config TFM_FLASH_NS_APP_WITH_TFM_BINARY
```

I suggest something shorter
```suggestion
config TFM_FLASH_MERGED_BINARY
```

---

Agreed, done.